frontend/pages/login.spec.js

```diff
@@ -0,0 +1,182 @@
+import { shallowMount } from "@vue/test-utils";
+import Login from "./login.vue";
+
+const stubs = ["BaseLoading", "brand-logo", "geometric-shape-a", "base-button"];
+
+const validAuthToken = btoa(
+  JSON.stringify({ username: "USERNAME", password: "PASSWORD" })
```

Comment:
A question here, not sure if make sense to align the expected format with the [Basic auth format](https://datatracker.ietf.org/doc/html/rfc7617#autoid-3):

> To receive authorization, the client
> 
>    1.  obtains the user-id and password from the user,
> 
>    2.  constructs the user-pass by concatenating the user-id, a single
>        colon (":") character, and the password,
> 
>    3.  encodes the user-pass into an octet sequence (see below for a
>        discussion of character encoding schemes),
> 
>    4.  and obtains the basic-credentials by encoding this octet sequence
>        using Base64 ([[RFC4648], Section 4](https://datatracker.ietf.org/doc/html/rfc4648#section-4)) into a sequence of US-ASCII
>        characters ([[RFC0020](https://datatracker.ietf.org/doc/html/rfc0020)]).

 instead of an encoded custom json one. Just to simplify the usability

Reply:
Yes, why not. I was thinking in a token a bit more "safe" although your proposal is simpler. I don't have any problem to update it.